pyproject.toml

```diff
@@ -77,9 +77,10 @@ doc = [
   "ipyleaflet",
   "colorama",
 ]
-test = ["pytest", "pytest-cov", "pytest-regressions"]
-dev = ["pyyaml", "pre-commit", "nox", "pydata-sphinx-theme[doc,test]"]
+test = ["pytest", "pytest-cov", "pytest-regressions", 'sphinx[test]']
+dev = ["pyyaml", "pre-commit", "pydata-sphinx-theme[doc,test]", "tox", "pandoc", "sphinx-theme-builder[cli]"]
 a11y = ["pytest-playwright"]
+i18n = ["Babel", "jinja2"]
```

Comment:
FWIW these are already dependencies of Sphinx itself:

https://github.com/sphinx-doc/sphinx/blob/ac3f74a3e0fbb326f73989a16dfa369e072064ca/pyproject.toml#L65-L69

Seems very unlikely sphinx will ever drop those deps, so I'd be inclined to remove this line. But I acknowledge that it also does no harm to leave it in, so your call @trallard

Reply:
Hmmm I added them because when I was trying to figure out how translations worked I kept getting errors about Jinja missing until I explicitly added them. 
Not sure if this is because I do not fully understand how translations work but having them as explicit dependencies helped. I can further assess later when I tackle the translation work again.